packages/editor/src/components/parts/program-start/activity/StartPart.tsx

```diff
@@ -0,0 +1,66 @@
+import { IVY_EXCEPTIONS, IVY_SCRIPT_TYPES, ProgramInterfaceStartData } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../../editors';
+import { ExceptionSelect, PathCollapsible, PathFieldset } from '../../common';
+import { useProgramInterfaceStartData } from './useProgramInterfaceStartData';
+import { ScriptInput, useFieldset } from '../../../../components/widgets';
+
+import { useValidations } from '../../../../context';
+import JavaClassSelector from '../JavaClassSelector';
+
+export function useProgramInterfaceStartPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useProgramInterfaceStartData();
+  const compareData = (data: ProgramInterfaceStartData) => [data.javaClass, data.exceptionHandler, data.timeout];
+  const validation = [...useValidations(['timeout']), ...useValidations(['exceptionHandler']), ...useValidations(['javaClass'])];
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return {
+    name: 'Start',
+    state,
+    reset: { dirty, action: () => reset() },
+    content: <StartPart />
+  };
+}
+
+const StartPart = () => {
+  const { config, update, updateTimeout } = useProgramInterfaceStartData();
+
+  const errorFieldset = useFieldset();
+  const timeoutSeconsFieldset = useFieldset();
+  const timeoutErrorFieldset = useFieldset();
+
+  return (
+    <>
+      <JavaClassSelector config={config.javaClass} onChange={change => update('javaClass', change)} type='ACTIVITY' />
+
+      <PathCollapsible label='Program' path='exceptionHandler' defaultOpen={true}>
+        <PathFieldset label='Error' path='error' {...errorFieldset.labelProps}>
+          <ExceptionSelect
+            value={config.exceptionHandler}
+            onChange={change => update('exceptionHandler', change)}
+            staticExceptions={[IVY_EXCEPTIONS.programException, IVY_EXCEPTIONS.ignoreException]}
+            inputProps={errorFieldset.inputProps}
+          />
+        </PathFieldset>
+      </PathCollapsible>
+
+      <PathCollapsible label='Timeout' path='timeout' defaultOpen={config.timeout !== undefined && config.timeout.seconds.length > 0}>

```

Comment:
better compare the config with the default config. Then it will still work if these defaults change sometimes and you also cover the error not only the seconds.